browser_use/browser/dolphin_service.py

```diff
@@ -98,7 +98,6 @@ async def switch_to_tab(self, page_id: int) -> None:
 
 		# Set the current page to the selected tab
 		self.page = self._pages[page_id]
-		await self.page.bring_to_front()  # Bring tab to the front
```

Comment:
unless I am missing something this is already called by `BrowserContext` 

https://github.com/browser-use/browser-use/blob/2948fc4889e38d0eb9f2a2c0e68cc3c9cab7e430/browser_use/browser/context.py#L404